blog/03-15-2023-how-to-deploy-a-celo-composer-application-on-spheron-protocol(decentralized-cloud-storage)/how-to-deploy-a-celo-composer-application-on-spheron-protocol(decentralized-cloud-storage).md

```diff
@@ -0,0 +1,245 @@
+---
+title: How to deploy a celo composer application on spheron protocol(decentralized cloud storage).
+description: In this tutorial, you will learn how to deploy a dapp built using celo composer on a decentalised cloud service called spheron protocol.
+authors:
+  - name: ✍️ Dennis Kimathi
+  - url: https://github.com/DennohKim
+  - image_url: https://avatars.githubusercontent.com/u/63696529?v=4
+tags: ["intermediate", "composer", "dapp", "dappstarter", "deploy"]
+hide_table_of_contents: true
+slug: /tutorial/how-to-deploy-a-celo-composer-application-on-spheron-protocol
+---
+
+![header](../../src/data-tutorials/showcase/intermediate/How_to_deploy_a_Celo_Composer_application_on_Spheron_protocol.png)
+
+
+## Introduction
+
+In this tutorial, we will be learning how to deploy a dapp built using celo composer on a decentalised cloud service called spheron protocol.
+
+
+## Prerequisites​
+
+To start building, you’ll need a basic understanding of web development, Node (v12), yarn, and Git.
+
+- [Node (v12)](https://nodejs.org/en/), [NVM](https://github.com/nvm-sh/nvm)
+- [Yarn](https://classic.yarnpkg.com/en/)
+- [Git](https://git-scm.com/)
+
+  
+## Celo Composer
+
+The easiest way to get started with Celo Composer is using `@celo/celo-composer`
+. This CLI tool enables you to quickly start building dApps on Celo for multiple frameworks including React, React Native (w/o Expo), Flutter, and Angular. You can create the dApp using the default Composer templates provided by Celo. In our case, we will work with react, specifically NextJS. The stack included in celo composer:
+
+- [NextJS](https://nextjs.org/) - React framework
+- [Typescript](https://www.typescriptlang.org/)
+- [React Celo](https://github.com/celo-org/react-celo)
+- [TailwindCSS](https://tailwindcss.com/) for styling
+- [Hardhat](https://hardhat.org/) Smart contract development environment
+
+## Spheron Protocol
```

Comment:
```suggestion
## Spheron Protocol

```

Reply:
What exactly have you changed here?

Reply:
Added a extra line

Reply:
But compiling issue is in above comments where you have added - url: check above comment

Reply:
Noted. Changes have been made.